test/common/TestBed.hpp

```diff
@@ -10,6 +10,9 @@
 #include "TestBedChild.hpp"
 #include "EnvVars.hpp"
 #include <gtest/gtest.h>
+#ifndef _GLIBCXX_IOMANIP
+#include <iomanip>
+#endif
```

Comment:
The blocker to prevent re-inclusion is already present inside the header. This is not necessary.
```suggestion
#include <iomanip>
```